website/common/locales/en/settings.json

```diff
@@ -204,11 +204,13 @@
     "transaction_gift_send": "Gifted to",
     "transaction_gift_receive": "Received from",
     "transaction_create_challenge": "Created challenge",
+    "transaction_create_bank_challenge": "Created bank challenge",
```

Comment:
I've never heard this term "bank challenge"--based on the rest of the code, it looks like that's when a Challenge is created that draws Gems from Guild balance?

Reply:
yes thats a challenge that is "payed" by the guild'S balance

@Tressley maybe a different sentence is needed or some kind of help label ?

Reply:
With the filtering so this only shows up for user support folks, we're free to adjust the language at will. ok as is!